sky/client/sdk.py

```diff
@@ -212,13 +213,16 @@ def list_accelerator_counts(
 @annotations.client_api
 def optimize(
     dag: 'sky.Dag',
-    minimize: common.OptimizeTarget = common.OptimizeTarget.COST
+    minimize: common.OptimizeTarget = common.OptimizeTarget.COST,
+    admin_policy_request_options: Optional[admin_policy.RequestOptions] = None
 ) -> server_common.RequestId:
     """Finds the best execution plan for the given DAG.
 
     Args:
         dag: the DAG to optimize.
         minimize: whether to minimize cost or time.
+        admin_policy_request_options: Request options used for admin policy
+            validation.
```

Comment:
Want to make this super clear that in most cases, this is not required. How about we add the following?
```suggestion
        admin_policy_request_options: Request options used for admin policy
            validation. This is only required when a admin policy is specified, see
            <link to admin policy>
```